cypress/integration/listingFlows/deleteListing.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Delete listing', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.viewport('macbook-16');
+
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.get('@user').then((user) => {
+      cy.loginUser(user).then(() => {
+        cy.createListing({
+          title: 'Test Listing',
+          content: `This is a test listing's contents.`,
+        }).then(() => {
+          cy.visit('/listings/dashboard');
+        });
+      });
+    });
+  });
+
+  it('deletes a listing', () => {
+    cy.findByText('Delete').click();
```

Comment:
```suggestion
    cy.findByRole('link', { name: 'Delete' }).click();
```

Just a suggestion, but `findByRole` is usually preferable as it's a bit more robust/specific than finding by text